### PR TITLE
Upgrade LiveEventWithDVR to track 2 SDK.

### PR DESCRIPTION
**Pull request by sxian0613**
Upgrade LiveEventWithDVR to track 2 SDK.

### CODE REVIEW

File: LiveIngest/LiveEventWithDVR/pom.xml [hunk 1]
```diff
@@ -14,36 +14,63 @@
...
+    </dependency>
+    <dependency>
+      <groupId>io.jsonwebtoken</groupId>
+      <artifactId>jjwt-api</artifactId>
+      <version>0.10.5</version>
+    </dependency>
+    <dependency>
+      <groupId>io.jsonwebtoken</groupId>
+      <artifactId>jjwt-impl</artifactId>
+      <version>0.10.5</version>
```
Comment (giakas) [inline LiveIngest/LiveEventWithDVR/pom.xml:65]: why do we need these extra dependencies?

File: LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java [hunk 2]
```diff
@@ -89,17 +76,17 @@ private static void runLiveEvent(ConfigWrapper config) {
             // Note: When creating a LiveEvent, you can specify allowed IP addresses in one of the following formats:                 
             //       IpV4 address with 4 numbers
             //       CIDR address range
-            IPRange allAllowIPRange = new IPRange().withName("AllowAll").withAddress("0.0.0.0").withSubnetPrefixLength(0);
-            List<IPRange> listIPRanges = new ArrayList<>();
+            IpRange allAllowIPRange = new IpRange().withName("AllowAll").withAddress("0.0.0.0").withSubnetPrefixLength(0);
```
Comment (johndeu) [inline LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java:79]: Might look nicer to move each with to it's own line here.

File: LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java [hunk 3]
```diff
@@ -89,17 +76,17 @@ private static void runLiveEvent(ConfigWrapper config) {
...
 
             // Create the LiveEvent input IP access control.
             LiveEventInputAccessControl liveEventInputAccess = new LiveEventInputAccessControl();
-            liveEventInputAccess.withIp(new IPAccessControl().withAllow(listIPRanges));
+            liveEventInputAccess.withIp(new IpAccessControl().withAllow(listIPRanges));
 
             // Create the LiveEvent Preview IP access control
             LiveEventPreview liveEventPreview = new LiveEventPreview();
-            liveEventPreview.withAccessControl(new LiveEventPreviewAccessControl().withIp(new IPAccessControl().withAllow(listIPRanges)));
+            liveEventPreview.withAccessControl(new LiveEventPreviewAccessControl().withIp(new IpAccessControl().withAllow(listIPRanges)));
```
Comment (johndeu) [inline LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java:89]: same, move the .with fluid statements to their own lines consistently.

File: LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java (hunk 2 above)
Comment (sxian0613) [reply LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java]: Move the .with fluid statements to their own lines consistently.

File: LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java (hunk 3 above)
Comment (sxian0613) [reply LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java]: Move the .with fluid statements to their own lines consistently.

**Review by johndeu — APPROVED** [review]

**Pull request merged**